third_party/move/move-compiler-v2/src/pipeline/control_flow_graph_simplifier.rs

```diff
@@ -454,6 +480,7 @@ impl ControlFlowGraphCodeGenerator {
                 acc
             },
         );
+        // check invariant 2
         assert!(self.successors.keys().collect_vec() == pred_map_reversed.keys().collect_vec());
```

Comment:
Similar changes as above in comparing iterators using `eq()` without having to explicitly collect.

Reply:
done